rt_mcca.m

```diff
@@ -109,15 +109,16 @@
 unmixing_flipped = unmixing;
 for k = 1:nstory
   fprintf('aligning polarity across channels for story %d/%d\n', k, nstory);
-  input = reshape(mixing(:,:,:,k), size(mixing,1), []);
```

Comment:
I am wondering about the reason to prefer the unmixing matrix over the mixing matrix for input definition. Why is the unmixing matrix a better candidate?

Reply:
this has been discussed